content-src/lib/selectLayoutRender.js

```diff
@@ -2,18 +2,29 @@ export const selectLayoutRender = (state, rickRollCache) => {
   const {layout, feeds, spocs} = state;
   let spocIndex = 0;
 
+  // rickRollCache stores random probability values for each spoc position. This cache is empty
+  // on page refresh and gets filled with random values on first render inside maybeInjectSpocs.
+  const isFirstRun = !rickRollCache.length;
```

Comment:
:+1:

Reply:
I think the buffer array could be stored in this local scope here, and not in base.

Reply:
done